src/services/Users.php

```diff
@@ -831,10 +845,11 @@ public function suspendUser(User $user): bool
      * Unsuspends a user.
      *
      * @param User $user The user.
+     * @param User|null $moderatingUser The user who is performing the action.
      * @return bool Whether the user was unsuspended successfully.
      * @throws \Throwable if reasons
      */
-    public function unsuspendUser(User $user): bool
+    public function unsuspendUser(User $user, ?User $moderatingUser = null): bool
     {
```

Comment:
Defaults to null to be non-breaking, and you can suspend a user outright by just not passing second arg.